configure-ironic.sh

```diff
@@ -123,11 +123,19 @@ EOF
     fi
 fi
 
+add_options_to_j2_config () {
+  if [[ "${IRONIC_TLS_SETUP}" == "true" ]]
+  then
+  crudini --set "/etc/ironic/ironic.conf.j2" Database connection "mysql+pymysql://ironic:{{ env.MARIADB_PASSWORD }}@127.0.0.1/ironic?charset=utf8&ssl=on&ssl_ca=/certs/ca/ironic/tls.crt"
```

Comment:
I don't think using crudini here is the correct way to do it. Jinaj2 is a templating language, so you can just use an if/else structure within the ironic.conf.j2, rather than this